package.json

```diff
@@ -1,11 +1,19 @@
 {
   "name": "preact-ssr-prepass",
-  "version": "1.1.3",
+  "version": "1.2.3",
   "description": "react-ssr-prepass drop-in replacement for preact-x",
   "main": "dist/index.js",
   "module": "dist/index.module.js",
   "umd:main": "dist/index.umd.js",
   "source": "src/index.js",
+  "exports": {
+		".": {
```

Comment:
I think you'll need to set your editor to "two spaces", should enforce this with an editorconfig/prettier 😅

Reply:
Someday 🙃

Fixed!